web/modules/custom/nlc_prototype/src/Form/DirectoryTokenAccessForm.php

```diff
@@ -78,27 +78,40 @@ public function access() {
    * {@inheritDoc}
    */
   public function buildForm(array $form, FormStateInterface $form_state) {
-    $format = filter_default_format();
     $form['intro'] = [
-      '#type' => 'processed_text',
-      '#text' => $this->t('We will email you a secure link so you can access the directory. The link will be active for 1 day.'),
```

Comment:
The text below has lost the message that the link is only active for 1 day. Is that not necessary because other text changes make it clear that the link will expire if it's not used?

Reply:
Aah, I think I can see the '1 day' part has been added to the 'outro' below. Am I right?

Reply:
Yep that's right